patches/@nrk+core-toggle+3.0.8.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/node_modules/@nrk/core-toggle/jsx.js b/node_modules/@nrk/core-toggle/jsx.js
```

Comment:
Hvorfor endring i denne fila?

Reply:
Var en funksjon som var patcha inn to ganger. Gjorde at man ikke fikk kjørt opp dev-eksemplene, så fiksa det i en egen commit (006a8b3)